### DISCUSSION TITLE
wrappid user lifecycle

### DISCUSSION THREAD

**Discussion by techoneel** [wrappid/.github#29] [Ideas]
# Wrappid User Lifecycle

This discussion thread is created for the ideas on how our final product helps users(developer/non-developer) to create an application and customise it through app-builder or manually write code.

## Non Developer

**Comment by anantakumarghosh** [comment]
> # Wrappid User Lifecycle

🤨 Can you please elaborate? @techoneel

**Reply by techoneel** [reply]
I hope you will understand now @anantakumarghosh

**Reply by anantakumarghosh** [reply]
> I hope you will understand now @anantakumarghosh

Yep, updated content helps a lot. :smiley:
